ci/pipelines/Build.jenkinsfile

```diff
@@ -46,25 +46,23 @@ pipeline
-
-							sh "mkdir build-${configurations[i]}"
-							dir("build-${configurations[i]}")
-							{
-								sh "conan install .. -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion}"
-								sh "cmake .. -G '${props.cmakeVS}' -A ${props.cmakePlatform}"
-								sh "cmake --build . --config ${props.cmakeConfiguration}"
-								sh "conan export-pkg ../conanfile.py RESTAPICore/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion} --force"
-								sh "conan test ../test_package/conanfile.py RESTAPICore/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion}"
-								sh "conan export-pkg ../test/RESTAPICoreTestUtilities/conanfile.py RESTAPICoreTestUtilities/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion} --force"
-								sh "conan test ../test/RESTAPICoreTestUtilities/test_package/conanfile.py RESTAPICoreTestUtilities/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion}"
-								sh "bin/${props.conanBuildType}/RESTAPICoreTest.exe --gtest_output=xml:RESTAPICoreTest.xml"
+					archs.each
+						{ arch ->
+							configs.each
+							{ config ->
+								stage("Build ${config}|${arch}")
+								{
+									def buildFolder = "build/${config}-${arch}"
+									bat "conan install . --install-folder ${buildFolder} --profile=${profile} -s arch=${arch} -s build_type=${config}"
+									bat "conan build . --build-folder ${buildFolder}" 
+									bat "conan export-pkg . RESTAPICore/${version}@systelab/${channel} --build-folder ${buildFolder} --force"
+									bat "conan test ./test_package/conanfile.py RESTAPICore/${version}@systelab/${channel} --profile=${profile} -s arch=${arch} -s build_type=${config}"
+									bat "conan export-pkg ./test/RESTAPICoreTestUtilities/conanfile.py RESTAPICoreTestUtilities/${version}@systelab/${channel} --build-folder ${buildFolder} --force"
+									bat "conan test ./test/RESTAPICoreTestUtilities/test_package/conanfile.py RESTAPICoreTestUtilities/${version}@systelab/${channel} --profile=${profile} -s arch=${arch} -s build_type=${config}"
+									dir("${buildFolder}/bin/${config}")
+									{
+										bat "RESTAPICoreTest.exe --gtest_output=xml:JWTUtilsTest.xml"
```

Comment:
Copy-paste error, use the same name as the UT project for the output xml.
Also, move this portion near the build of the library since is where the UT will be build too.

Reply:
oopsie.. I'll change it. Thanks